servers/rendering/renderer_rd/pipeline_hash_map_rd.h

```diff
@@ -77,9 +77,16 @@ class PipelineHashMapRD {
 	}
 
 	void _wait_for_all_pipelines() {
-		MutexLock local_lock(local_mutex);
-		for (KeyValue<uint32_t, WorkerThreadPool::TaskID> key_value : compilation_tasks) {
-			WorkerThreadPool::get_singleton()->wait_for_task_completion(key_value.value);
+		Vector<WorkerThreadPool::TaskID> tasks_to_wait;
+		{
+			MutexLock local_lock(local_mutex);
+			for (KeyValue<uint32_t, WorkerThreadPool::TaskID> key_value : compilation_tasks) {
+				tasks_to_wait.append(key_value.value);
```

Comment:
```suggestion
				tasks_to_wait.push_back(key_value.value);
```
LocalVector doesn't have `append()` so this needs to change too